src/mapValues.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Maps values of `object` and keeps the same keys.
+ * @param object the object to map
+ * @param fn the mapping function
+ * @signature
+ *    R.mapValues(object, fn)
+ * @example
+ *    R.mapValues({a: 1, b: 2}, (value, key) => value + key) // => { a: 1a, b: 2b }
```

Comment:
quotes are missing in `{ a: 1a, b: 2b }`

Reply:
did that here: 7e309318

I also removed the spaces around the brackets, so that it is the same formatting as in other examples.